docs/contributors_agreement_explained.rst

```diff
@@ -1,7 +1,6 @@
 .. -*- coding: utf-8 -*-
-
 ===========================================
-Contributor’s Agreement for Plone Explained
+Contributor’s Agreement For Plone Explained
```

Comment:
prepositions in titles are not usually capitalized

Reply:
There are, please check our style guide

Reply:
OK. That goes against what I was taught, but all right.

Reply:
Why ?
We have that in our style-guide and slowly changing lots of headings.

In the world of technical writing there are tons of opinions about how headings should look like.

There are even tons of talks about that topic on every ' docs conference'.

Google is doing it different than Facebook and FB is doing it different than company X.

There is currently no right or wrong.

There a valid reasons for lots of different ways on how to to it.

The important point is if you decide on one way use it through the whole docs. And this is exactly what we are doing/improving now.